src/main/java/com/ably/kafka/connect/client/AblyClient.java

```diff
@@ -0,0 +1,13 @@
+package com.ably.kafka.connect.client;
+
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.errors.RetriableException;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+public interface AblyClient {
+    void connect() throws ConnectException;
+    void publishFrom(SinkRecord message) throws ConnectException, RetriableException;
```

Comment:
In the `DefaultAblyClient` you've renamed this method param to `record` and I think it fits better than `message`. Please rename this method param to `record` :wink:

Reply:
This is a good point 👍  https://github.com/ably/kafka-connect-ably/pull/63/commits/d9411ee8a1d1ad0fc2c2a317b32294ec1e9a399f